docker/package/model.py

```diff
@@ -17,6 +17,8 @@
     Unit,
 )
 
+letter_version = "a"
```

Comment:
Can you add a comment (or perhaps a clearer name?) just so we clarify what this is for?

I also think we should mention this in our release template now, as we should reset it to `""` whenever we bump the native release version(s)

Reply:
Perhaps, it also can be moved to a `TezosBakingServicesPackage` class definition